apps/aehttp/src/aehttp_dispatch_ext.erl

```diff
@@ -212,6 +214,86 @@ handle_request('PostOracleResponse', #{'OracleResponseTx' := Req}, _Context) ->
                                                  aec_tx:serialize_to_binary(Tx))}}
     end;
 
+handle_request('PostNamePreclaim', #{'NamePreclaimTx' := Req}, _Context) ->
+    ParseFuns = [parse_map_to_atom_keys(),
+                 read_required_params([account, commitment, fee]),
+                 base58_decode([{account, account, account_pubkey},
+                                {commitment, commitment, commitment}]),
+                 get_nonce(account)
```

Comment:
Currently considering whether external ok.

Reply:
We should either do this or provide an endpoint for getting once's nonce (which would probably be the same...)